src/components/utility/CopyCodeToClipboard/helpers.ts

```diff
@@ -0,0 +1,34 @@
+export const createCopyButton = () => {
+	const button = document.createElement('button')
+	button.classList.add('copy-button')
+	button.innerHTML = 'Copy'
+	return button
+}
+
+export const createCopyButtonWrapper = () => {
+	const wrapper = document.createElement('div')
+	wrapper.classList.add('code-wrapper')
+	return wrapper
+}
+
+export const handleCopyButtonClick = async (
+	button: HTMLButtonElement,
+	text: string
+) => {
+	const buttonTextsAfterClick = ['Copied', 'Error']
```

Comment:
You don't need to add a "button" prefix, since this function name already says this is for the button element.

Reply:
This is just needlessly repeated.